Sources/BraintreeCore/BTShopperInsightsRequest.swift

```diff
@@ -4,25 +4,55 @@ import Foundation
 /// - Note: This feature is in beta. It's public API may change or be removed in future releases.
 public struct BTShopperInsightsRequest {
     
-    /// The buyer's email address.
-    private let email: String
+    // MARK: - Private Properties
     
-    /// The buyer's country code prefix to the national telephone number. An identifier for a specific country.
-    /// Must not contain special characters.
-    private let phoneCountryCode: String
+    private var email: String?
+    private var phone: Phone?
     
-    /// The buyer's national phone number. Must not contain special characters.
-    private let phoneNationalNumber: String
+    // MARK: - Initializers
+
+    /// Initialize a `BTShopperInsightsRequest`
+    /// - Parameters:
+    ///   - email: The buyer's email address.
+    ///   - phone: The buyer's phone number details.
+    /// - Note: This feature is in beta. It's public API may change or be removed in future releases.
+    public init(email: String, phone: Phone) {
+        self.email = email
+        self.phone = phone
+    }
     
     /// Initialize a `BTShopperInsightsRequest`
     /// - Parameters:
     ///   - email: The buyer's email address.
-    ///   - phoneCountryCode: The buyer's country code prefix to the national telephone number. An identifier for a specific country. Must not contain special characters.
-    ///   - phoneNationalNumber: The buyer's national phone number. Must not contain special characters.
     /// - Note: This feature is in beta. It's public API may change or be removed in future releases.
-    public init(email: String, phoneCountryCode: String, phoneNationalNumber: String) {
+    public init(email: String) {
         self.email = email
-        self.phoneCountryCode = phoneCountryCode
-        self.phoneNationalNumber = phoneNationalNumber
+    }
+    
+    /// Initialize a `BTShopperInsightsRequest`
+    /// - Parameters:
+    ///   - phone: The buyer's phone number details.
+    /// - Note: This feature is in beta. It's public API may change or be removed in future releases.
+    public init(phone: Phone) {
+        self.phone = phone
+    }
+    
+    // MARK: - Data Types
+    
+    /// Buyer's phone number details.
+    public struct Phone {
+        
+        private let phoneCountryCode: String
+        private let phoneNationalNumber: String
```

Comment:
```suggestion
        private let countryCode: String
        private let nationalNumber: String
```
nit: To make property accesses like `phone.countryCode` and `phone.nationalNumber` read cleanly.